test/mitmproxy/utils/test_typecheck.py

```diff
@@ -32,10 +32,9 @@ def test_check_union():
     with pytest.raises(TypeError):
         typecheck.check_option_type("foo", [], typing.Union[int, str])
 
-    # Python 3.5 only defines __union_params__
     m = mock.Mock()
     m.__str__ = lambda self: "typing.Union"
-    m.__union_params__ = (int,)
+    m.__args__ = (int,)
```

Comment:
The whole mock is not required anymore, that was just to test 3.5. (same below)

Reply:
I fiddled with it a bit, without much success... 
What do we have to set `m` to, to make it work?

Reply:
Just remove the whole mock part entirely, this was just to get test coverage without running the old version.